README.md

```diff
@@ -1,46 +1,192 @@
+    // configuration is correct for your production server
+    'X-Content-Type-Options': 'nosniff',
+    // Very new. Used to entirely disable browser features like geolocation per host.
+    // Since we don't know what your site may need, we don't try to set this
+    // header by default (false means "don't send the header")
+    'Permissions-Policy': false,
+    // Don't send a "Referer" (sp) header unless the new URL shares the same
+    // origin. You can set this to `false` if you prefer cross-origin "Referer"
+    // headers be sent. Apostrophe does not rely on them
+    'Referrer-Policy': 'same-origin',
+    // `true` means it should be computed according to the rules below.
+    // You may also pass your own string, or `false` to not send this header.
+    // The `policies` option and all of its sub-options are ignored unless
+    // `Content-Security-Policy` is `true`.
+    'Content-Security-Policy': true
+  }
+};
+```
+
+For more information about these security headers see the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers#security).
+
+### Configuring the `Content-Security-Policy` header
+
+The `Content-Security-Policy` header is more complex than the others. As described [in MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy), this header is used to determine which hosts are permitted as a source for stylesheets, scripts, images and more. While you can certainly set a fixed value for it, the default response for it is the result of merging together options for individual use cases ("policies") as shown below. This makes it easier to think about what you need to allow for a particular purpose. The actual header sent contains all of the permissions required to satisfy all of the policies.
+
+The default configuration is shown below. You do **not** have to copy and paste the entire default configuration, only the policies you wish to change. Any policy you specify explicitly at project level overrides all of the default settings shown below for that
+policy. You may set one to `false` to completely disable it. You may also introduce entirely new policies.
+
+Policies of the same type from different sub-options are merged, with the largest set of keywords and hosts enabled. This is done because browsers do not support more than one style-src policy, for example, but do support specifying several hosts.
+
+Note the `HOSTS` wildcard which matches all expected hosts that Apostrophe is aware of, including `baseUrl` settings, CDN hosts and locale-specific hostnames.
+
+```javascript
+module.exports = {
+  // in modules/@apostrophecms/security-headers/index.js
+  options: {
+    policies: {
+      general: {
+        'default-src': 'HOSTS',
+        // Because it is necessary for some of the output of our rich text editor
+        'style-src': "HOSTS 'unsafe-inline'",
+        'script-src': 'HOSTS',
+        'font-src': 'HOSTS',
+        'frame-src': `'self'`
```

Comment:
Should this `'self'` jawn be in backticks?

Reply:
Switched to double quotes to preserve readability here (the single quotes are a real requirement in the content of the string).